utils/crypto/bls/signers/json-rpc/server.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package jsonrpc
+
+import (
+	"context"
+	"log"
+	"net"
+	"net/http"
+	"time"
+
+	"github.com/gorilla/rpc/v2"
+	"github.com/gorilla/rpc/v2/json"
+
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls/signers/localsigner"
+)
+
+type signerService struct {
+	signer *localsigner.LocalSigner
+}
+
+type Server struct {
+	httpServer *http.Server
+	listener   net.Listener
+}
+
+func NewSignerService() *signerService {
+	signer, err := localsigner.NewSigner()
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Can we bubble up the error here instead of panic'ing? I think for the other panics it's reasonable if we don't want to make the diff bigger because it's an interface change + the panic's wont trigger because the network signer isn't actually used.

Reply:
The constructor isn't actually part of the interface. Easy to add this, 👍